tests/mentorship_relation/test_dao_listing.py

```diff
@@ -45,9 +94,9 @@ def test_dao_list_mentorship_relation_accepted(self):
         db.session.add(self.mentorship_relation)
         db.session.commit()
 
-        result = DAO.list_mentorship_relations(user_id=self.first_user.id, accepted=True)
+        result = DAO.list_mentorship_relations(user_id=self.first_user.id, rel_states=["accepted"])
 
-        self.assertEqual((messages.NOT_IMPLEMENTED, 200), result)
+        self.assertEqual(([self.mentorship_relation, self.accepted_mentorship_relation], 200), result)
```

Comment:
@bartekpacia Why don't you just 
`assertEqual(([self.accepted_mentorship_relation],200), result)`
Why do you include `self.mentorship_relation` too in all the tests. 
It is a pending relation right?

Reply:
I think that more cases and input testing = better quality of tests

Reply:
No, that's not my doubt. My doubt is
This is idle thing
`AssertEqual(ACCEPTED,ACCEPTED)`

but what I see is
`AssertEqual([PENDING,ACCEPTED], [ACCEPTED])`

Maybe I am wrong somewhere, but that's what I am not able to figure out.

Reply:
Okay, I'll look into this.

Reply:
Okay, after some little investigation, I discovered that large part of the code I wrote for those test is unnecessary and can be nicely refactored.
Thank you!